cmd/temporalite/main.go

```diff
@@ -262,6 +269,21 @@ func buildCLI() *cli.App {
 				}
 				opts = append(opts, temporalite.WithLogger(logger))
 
+				configVals, err := getDynamicConfigValues(c.StringSlice(dynamicConfigValueFlag))
+				if err != nil {
+					return err
+				}
+				// If there is no config value for search attribute cache disabling,
+				// default it to true
```

Comment:
This extra bit of explanation from your PR description would be helpful here too IMO.

```suggestion
				// If there is no config value for search attribute cache disabling,
				// default it to true. This is needed in order to avoid a wait period
				// before search attributes are available for use.
```

Reply:
No longer applicable, this is no longer in CLI